nina.css

```diff
@@ -0,0 +1,132 @@
+.container {
+  background: url(/photos/header-shapes\ mobile.png)  no-repeat;
+  background-size: cover;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+}
+
+.header {
+  display: flex;
+  top: 44px;
+  bottom: 0.001px;
+  justify-content: space-between;
+}
+
+.header h1 {
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  margin-left: 14px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.title {
+  width: 85%;
+  line-height: 1.4em;
+}
+
+.title h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
```

Comment:
- [ ] Please add a margin-top value to this title to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/165502401-a296b33c-c26d-4365-a806-fd1a23c9209e.png)  | ![image](https://user-images.githubusercontent.com/68796232/165502686-d7064450-8ca8-4cda-a6c6-7cbeb47d0d6d.png) |